src/webgl/interaction.js

```diff
@@ -93,14 +93,18 @@ p5.prototype.orbitControl = function(sensitivityX, sensitivityY, sensitivityZ) {
 
   const zoomScaleFactor = 0.02;
   const scaleFactor = this.height < this.width ? this.height : this.width;
+  const uP = this._renderer.uPMatrix.mat4;
 
   // ZOOM if there is a change in mouseWheelDelta
   if (this._mouseWheelDeltaY !== 0) {
     // zoom according to direction of mouseWheelDeltaY rather than value
-    if (this._mouseWheelDeltaY > 0) {
-      this._renderer._curCamera._orbit(0, 0, sensitivityZ * zoomScaleFactor);
-    } else {
-      this._renderer._curCamera._orbit(0, 0, - sensitivityZ * zoomScaleFactor);
+    const mouseWheelSign = (this._mouseWheelDeltaY > 0 ? 1 : -1);
+    const deltaRadius = mouseWheelSign * sensitivityZ * zoomScaleFactor;
+    this._renderer._curCamera._orbit(0, 0, deltaRadius);
+    // if ortho, scale change.
```

Comment:
For future contributors, it's probably worth elaborating a little bit here about why this is necessary (ortho cameras don't make far things smaller, so just moving the camera wouldn't result in any visual change)

Reply:
Certainly, the projection matrix is ​​only manipulated when performing orthogonal projection, so an explanation is necessary...
For example, ”For orthogonal projection, the elements of the projection matrix must be changed so that the scaling is done.”?

Reply:
I think that's a good start, although it would also be good to explain why scaling is necessary at all for ortho cameras (so, why just moving the camera isn't sufficient)

Reply:
I see, I'm sure it would be easier to explain the reason for the change... how about this?: "In orthogonal projection, the scale does not change even if the distance to the gaze point is changed, so the projection matrix needs to be modified."

Reply:
sounds great, thanks!